app/controllers/cloud_object_store_object_controller.rb

```diff
@@ -5,16 +5,17 @@ class CloudObjectStoreObjectController < ApplicationController
   after_action :set_session_data
 
   include Mixins::GenericListMixin
+  include Mixins::GenericButtonMixin
   include Mixins::GenericSessionMixin
 
   def breadcrumb_name(_model)
     ui_lookup(:tables => "cloud_object_store_object")
   end
 
   def button
```

Comment:
On the other hand he `button` method here is the same as in `app/controllers/cloud_object_store_container_controller.rb`. That, I am sure, we don't want.